src/GeneratedInjectorDelegator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Di;
+
+use Psr\Container\ContainerInterface;
+
+class GeneratedInjectorDelegator
+{
+    /**
+     * @param string $name
+     */
+    public function __invoke(ContainerInterface $container, $name, callable $callback) : InjectorInterface
+    {
+        $config = $container->has('config') ? $container->get('config') : [];
+        $aotConfig = $config['dependencies']['auto']['aot'] ?? [];
+        $namespace = $aotConfig['namespace'] ?? null;
```

Comment:
`Zend\Di\CodeGenerator\InjectorGenerator` defaults the namespace to `'Zend\Di\Generated'` on [line 98](https://github.com/zendframework/zend-di/blob/develop/src/CodeGenerator/InjectorGenerator.php#L98), I think this should do the same here as well.

Reply:
Also disagree here, this is the namespace from the configuration, we are using later on. The only think we can change is to set it to empty string. It's not making any difference here: as we using it later in the concatenation.

Reply:
We have done here the same what is in the other place:

https://github.com/zendframework/zend-di/blob/7a686b016719069862d225c3dfb6d170646c99c8/src/Container/GeneratorFactory.php#L29-L33